src/utils.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+
+def compare_2_df(df1: pd.DataFrame, df2: pd.DataFrame) -> None:
+    """Compare two Pandas DataFrames and print the differences.
+
```

Comment:
This docstring is not precise enough, it compares only the common columns of the dataframes